R/pkg/R/generics.R

```diff
@@ -1403,20 +1416,25 @@ setGeneric("unix_timestamp", function(x, format) { standardGeneric("unix_timesta
 #' @export
 setGeneric("upper", function(x) { standardGeneric("upper") })
 
-#' @rdname var
+#' @rdname column_aggregate_functions
+#' @param y,na.rm,use currently not used.
```

Comment:
hmm, this is in `var` but also in `sd` too - perhaps should move to column_aggregate_functions

in fact, in your screen shot they are both listed https://github.com/apache/spark/pull/18025#issuecomment-303877945
`y,na.rm,use` and then `na.rm`